src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -254,19 +254,22 @@ public DataFrame Description()
             return ret;
         }
 
-        public DataFrame Sort(string columnName, bool ascending = true)
+        /// <summary>
+        /// Orders the data frame by a specified column.
```

Comment:
Does this sound better for the documentation here?:

`Sorts the rows of the dataframe in ascending order according to the values in <param name="columnName">`